app/src/org/commcare/views/EntityView.java

```diff
@@ -482,7 +482,8 @@ public static Spannable highlightSearches(String[] searchTerms, Spannable raw,
                             int indexInDisplay = normalizedDisplayString.indexOf(currentSpan);
                             int length = (curEnd - curStart);
 
-                            if (indexInDisplay != -1 && StringUtils.fuzzyMatch(currentSpan, searchText).first) {
+                            if (indexInDisplay != -1 &&
+                                    org.commcare.cases.util.StringUtils.fuzzyMatch(currentSpan, searchText).first) {

```

Comment:
It seems weird that this got moved to this line, rather than the import changing?